web/app/scripts/src/services/api.es.js

```diff
@@ -1,37 +1,41 @@
 import { angular } from '../libraries';
 
-import ProjectConfig from '../providers/project-config';
+import ProjectConfigModule from '../providers/project-config';
```

Comment:
I thought it's going to be confusing with the actual `ProjectConfig` used in the service